modules/microshift-check-journal-logs-updates.adoc

```diff
@@ -0,0 +1,41 @@
+//Module included in the following assemblies:
+//
+//* microshift_troubleshooting/microshift-updates-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-check-journal-logs-updates_{context}"]
+= Checking {product-title} journal logs after updates
+
+In some cases, {product-title} might fail to update. In these events, it is helpful to understand failure types and how to troubleshoot them. The journal logs can assist in diagnosing update failures.
+
+.Procedure
+
+. Check the {product-title} journal logs by running the following command:
++
+[source,terminal]
+----
+$ sudo journalctl -u microshift
+----
+
+. Check the Greenboot journal logs by running the following command:
++
+[source,terminal]
+----
+$ sudo journalctl -u greenboot-healthcheck
+----
+
+. List the boot journal logs by running the following command:
++
+[source,terminal]
+----
+$ sudo journalctl --list-boots
+----
+
+. Check the boot journal logs by using the following command syntax:
++
+[source,terminal]
+----
+$ sudo journalctl --boot <-3>
```

Comment:
This command can also get `-u` option to get logs of a service from a boot. Otherwise it will dump whole journal of a particular boot which might be too much